pkg/runtime/runtime.go

```diff
@@ -1031,10 +1031,10 @@ func (a *DaprRuntime) initProxy() {
 	a.proxy = messaging.NewProxy(messaging.ProxyOpts{
 		AppClientFn:        a.grpc.GetAppClient,
 		ConnectionFactory:  a.grpc.GetGRPCConnection,
-		AppID:              a.runtimeConfig.ID,
+		AppID:              a.runtimeConfig.id,
 		ACL:                a.accessControlList,
 		Resiliency:         a.resiliency,
-		MaxRequestBodySize: a.runtimeConfig.MaxRequestBodySize,
+		MaxRequestBodySize: a.runtimeConfig.maxRequestBodySize,
 	})
 
 	log.Info("gRPC proxy enabled")
```

Comment:
I don't think we need this either at this point (in the past, gRPC proxying was a feature gated behind a flag, but now it's not)